Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -148,6 +153,25 @@ namespace AzToolsFramework
         {
         }
 
+        void AssetBrowserTableView::mouseMoveEvent(QMouseEvent* mouseEvent)
+        {
+            QTableView::mouseMoveEvent(mouseEvent);
+
+            QModelIndex hoveredIndex = indexAt(mouseEvent->pos());
+            int oldHeveredRow = m_hoveredRow;
+
+            m_hoveredRow = hoveredIndex.row();
+            m_hoveredColumn = hoveredIndex.column();
+
+            if (selectionBehavior() == SelectRows && oldHeveredRow != m_hoveredRow)
+            {
+                for (int i = 0; i < model()->columnCount(); ++i)
+                {
+                    update(model()->index(m_hoveredRow, i));
```

Comment:
suggestion: A guide comment might be useful here to indicate why this is needed and what it's doing exactly, I take it this is just refreshing the highlight?